gensim/test/test_keyedvectors.py

```diff
@@ -255,6 +255,15 @@ def test_add_multiple(self):
         for ent, vector in zip(entities, vectors):
             self.assertTrue(np.allclose(kv[ent], vector))
 
+    def test_add_type(self):
+        kv = EuclideanKeyedVectors(2)
```

Comment:
`EuclideanKeyedVectors` is a nonsensical alias to `KeyedVectors` based on some deprecated misfactoring, so shouldn't be used.

Reply:
Fixed 38daff5 (just get rid alias from import)